### ISSUE TITLE
Thank you for your work at CVPR24

### ISSUE THREAD

**Issue by chenrxi** [HongxiaLee/FedOTP#1]
Could you provide the link to your paper as I'm interested in your work and by the way, you ignore the feature of classification token to classify the image?

**Comment by chenrxi** [comment]
and the logits has nan value at the beginning
